### PR TITLE
Mediated transfer states

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

**Comment by LefterisJP** [comment]
But you did not fix all spelling mistakes @palango ;) 
Anyway merge it, we can always fix typos later.

**Comment by palango** [comment]
Damn, what did I miss?

File: raiden/transfer/mediated_transfer/state.py [hunk 1]
```diff
@@ -28,6 +28,171 @@ def lockedtransfer_from_message(message):
...
+        if not isinstance(transfer_description, TransferDescriptionWithSecretState):
+            raise ValueError(
+                'transfer_description must be an instance of TransferDescriptionWithSecretState'
+            )
+
+        # This is the users description of the transfer. It does not contain a
+        # balance proof and it's not related to any channel.
+        self.transfer_description = transfer_description
+
+        # This it the channel used to satisfy the above transfer.
```
Comment (LefterisJP) [inline raiden/transfer/mediated_transfer/state.py]: @palango this

**Comment by LefterisJP** [comment]
@hackaugusto Since these are not used by a test I will kill the build and merge asap. No need to waste 30 mins with Travis.

**Pull request merged**